pallets/ddc-staking/src/lib.rs

```diff
@@ -861,6 +866,46 @@ pub mod pallet {
 			<Pricing<T>>::set(Some(price_per_byte));
 			Ok(())
 		}
+
+		/// Add a new account to the list of cluster managers.
+		///
+		/// RuntimeOrigin must be Root to call this function.
+		#[pallet::weight(T::WeightInfo::allow_cluster_manager())]
+		pub fn allow_cluster_manager(
+			origin: OriginFor<T>,
+			grantee: <T::Lookup as StaticLookup>::Source,
+		) -> DispatchResult {
+			ensure_root(origin)?;
```

Comment:
Do want to have the root defining cluster managers in mainnet as well?
Maybe there should be some governance process

Reply:
Please check similar thread here: https://github.com/Cerebellum-Network/blockchain-node/pull/82#discussion_r1312721846